### DISCUSSION TITLE
How to replace ar-prompt to my own image (fixed in the center of screen, without animations)!

### DISCUSSION THREAD

**Discussion by Cebolitoss** [google/model-viewer#4038] [Q&A] [answered]
I'm trying to replace the hand.png to my own .gif. I could change it setting the src in the img input of the "ar-prompt"!

But it keeps going around in circles, in the way that the hand.png does.
I tried using:

        #ar-prompt img{
            animation: none !important;
        }

But it didn't help. How can I do that? 
I just need to replace the actual png for my own gif.
But the .png that is animated and keep doing a circle movement, and I don't want the .gif to make the same.
The gif should be fixed in the middle of the screen.

What do I do?

I found it is a bit hard to debug this feature (ar-prompt) on the browser, considering it doesn't run AR!
How can I debug this feature (ar-prompt)? I mean the png/command that says to the user "move the device to find a surface".

**Comment by elalish** [comment, marked as answer]
It's a little hard to tell without code or a link, but I'm guessing you're modifying this [example](https://modelviewer.dev/examples/augmentedreality/index.html#webXR)? This line causes the circle animation, so you can just remove that from the CSS: `animation: elongate 2s infinite ease-in-out alternate;`.

For debug, simply connect your Android phone running Chrome to your laptop over USB, and point your laptop Chrome to chrome://inspect. Accept permissions and you can see and debug your Android chrome tabs on your laptop. You can even forward ports to let your phone access a local web server. Then you really can debug in AR.
